x/params/client/cli/tx.go

```diff
@@ -0,0 +1,74 @@
+package cli
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/utils"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtxb "github.com/cosmos/cosmos-sdk/x/auth/client/txbuilder"
+	"github.com/cosmos/cosmos-sdk/x/gov"
+	"github.com/cosmos/cosmos-sdk/x/params"
+	paramscutils "github.com/cosmos/cosmos-sdk/x/params/client/utils"
+)
+
+// GetCmdSubmitProposal implements a command handler for submitting a parameter
+// change proposal transaction.
+func GetCmdSubmitProposal(cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "param-change [proposal-file]",
+		Args:  cobra.ExactArgs(1),
+		Short: "Submit a parameter change proposal",
+		Long: strings.TrimSpace(`
+Submit a parameter proposal along with an initial deposit. The proposal details
+must be supplied via a JSON file.
+
+Example:
+$ gaiacli tx gov submit-proposal param-change <path/to/proposal.json> --from=<key_or_address>
+where proposal.json contains:
+{
+  "title": "Staking Param Change",
+  "description": "Update max validators",
+  "changes": [
+    {
+      "subspace": "staking",
+      "key": "MaxValidators",
+      "value": "105"
```

Comment:
I wonder if there is a convenient way to use a json value directly in the `value` field and later stringify. Currently the  parameter store is using json encoding only so it won't be a problem. So in this case the field value will be `105` not `"105"`.  Double encoded json value is annoying sometimes too. For example string encoded integer "100" has to be written as `"\"100\""`. It can be addressed in another PR.

Reply:
https://golang.org/pkg/encoding/json/#RawMessage seems like an option - we can leave the target struct's `value` field to `json.RawMessage` and use it directly as the `value string` in parameter change.

Reply:
@mossid can you open up a separate PR for this? I'd like to potentially address this outside of this PR as it's not blocking.